### PR TITLE
[4.0][RFC] Custom Element: Joomla! Toolbar button

### PR DESCRIPTION
**Pull request by Fedik**
This is redo of #19635 

This patch add `<joomla-toolbar-button/>` custom element, to standard toolbar button.
It allow us to move most of inline onclick="" to separate file.

Currently it only for `StandardButton`.

### Testing Instructions
Click the toolbar buttons here and there, and make sure all works as expected, as before the patch.

ping @dgrammatiko @wilsonge

### CODE REVIEW

**Comment by dgrammatiko** [comment]
@Fedik can you check https://github.com/Fedik/joomla-cms/pull/8

**Comment by Fedik** [comment]
@dgrammatiko thanks

**Comment by infograf768** [comment]
Did not remark any change in toolbar behavior after patch.

File: build/webcomponents/js/toolbar-button/toolbar-button.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        }
+      }
+    }
+
+    executeTask() {
+      if (this.disabled) {
+        return false;
+      }
+
+      if (this.confirmMessage && !confirm(this.confirmMessage)) {
```
[the remark this answers is not in the document]
Comment (Fedik) [reply build/webcomponents/js/toolbar-button/toolbar-button.js]: this was expected. deal with it :neckbeard:

[the remark this answers is not in the document]
Comment (wilsonge) [reply build/webcomponents/js/toolbar-button/toolbar-button.js]: told hound to deal with it :)

**Comment by wilsonge** [comment]
Awesome work! Thanks for being so patient with this one!

**Pull request merged**